<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="approximationsBox">
     <property name="title">
      <string>Approximations</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="rareEvent">
        <property name="text">
         <string>Rare-event</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mcub">
        <property name="text">
         <string>MCUB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="analysesBox">
     <property name="title">
      <string>Analyses</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="primeImplicants">
        <property name="text">
         <string>Prime implicants</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="probability">
        <property name="text">
         <string>Probability</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="importance">
        <property name="text">
         <string>Importance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="algorithmsBox">
     <property name="title">
      <string>Algorithms</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="mocus">
        <property name="text">
         <string>MOCUS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bdd">
        <property name="text">
         <string>BDD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="zbdd">
        <property name="text">
         <string>ZBDD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="limitsBox">
     <property name="title">
      <string>Limits</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMissionTime">
        <property name="text">
         <string>Mission time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="missionTime">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelProductOrder">
        <property name="text">
         <string>Product order:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="productOrder"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>primeImplicants</tabstop>
  <tabstop>probability</tabstop>
  <tabstop>importance</tabstop>
  <tabstop>mocus</tabstop>
  <tabstop>bdd</tabstop>
  <tabstop>zbdd</tabstop>
  <tabstop>approximationsBox</tabstop>
  <tabstop>rareEvent</tabstop>
  <tabstop>mcub</tabstop>
  <tabstop>missionTime</tabstop>
  <tabstop>productOrder</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
